exo-go/src/app_config_helpers/index.go

```diff
@@ -0,0 +1,54 @@
+package appConfigHelpers
+
+import (
+	"bufio"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+
+	"github.com/Originate/exosphere/exo-go/src/types"
+	"github.com/Originate/exosphere/exo-go/src/user_input_helpers"
+	"gopkg.in/yaml.v2"
+)
+
+// GetAppConfig reads application.yml and returns the appConfig object
+func GetAppConfig() types.AppConfig {
+	yamlFile, err := ioutil.ReadFile("application.yml")
+	if err != nil {
+		log.Fatalf("Failed to read application.yml: %s", err)
+	}
+	var appConfig types.AppConfig
+	err = yaml.Unmarshal(yamlFile, &appConfig)
+	if err != nil {
+		log.Fatalf("Failed to unmarshal application.yml: %s", err)
+	}
+	return appConfig
+}
+
+// UpdateAppConfig adds serviceRole to the appConfig object and updates
+// application.yml
+func UpdateAppConfig(serviceRole string, appConfig types.AppConfig) {
+	reader := bufio.NewReader(os.Stdin)
+	switch protectionLevel := userInputHelpers.Choose(reader, "Protection Level:", []string{"public", "private"}); protectionLevel {
```

Comment:
do you need the `protectionLevel` variable? I think this could be
```go
switch userInputHelpers.Choose(reader, "Protection Level:", []string{"public", "private"}) {
```